src/components/structures/MatrixChat.js

```diff
@@ -1118,6 +1118,7 @@ module.exports = React.createClass({
             const payload = {
                 action: 'view_room',
                 event_id: eventId,
+                // If an event ID is set (truthy), mark it as highlighted
```

Comment:
well, now it has a comment, but I'm not sure I'm any the wiser - the comment says what the code is doing (which I could see for myself), but not *why*.